src/python/grpcio_tests/tests/unit/_signal_client.py

```diff
@@ -0,0 +1,60 @@
+"""Client for testing responsiveness to signals."""
+
+from __future__ import print_function
+
+import argparse
+import functools
+import logging
+import signal
+import sys
+
+import grpc
+
+SIGTERM_MESSAGE = "Handling sigterm!"
+
+UNARY_UNARY = "/test/Unary"
+UNARY_STREAM = "/test/ServerStreaming"
+
+_MESSAGE = b'\x00\x00\x00'
+
+_ASSERTION_MESSAGE = "Control flow should never reach here."
+
+
+def handle_sigint(rpc_future, unused_signum, unused_frame):
+    print(SIGTERM_MESSAGE)
```

Comment:
Should we use `logging`?

Reply:
Logging is for consumption by humans. This is intended to be consumed just by a machine.

Reply:
Then we can remove the unused `logging`?